src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,120 @@
 package co.inventorsoft.academy.collections.model;
 
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.Iterator;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Function;
 
 public class Range<T> implements Set<T> {
 
+
+    public List<T> MyList = new ArrayList<>();
```

Comment:
In java we name fields starting with lowercase

Reply:
Changed fields to lowercase